static_compress/mixin.py

```diff
@@ -71,47 +72,64 @@ def get_modified_time(self, name):
         return self._datetime_from_timestamp(getmtime(alt))
 
     def post_process(self, paths, dry_run=False, **options):
-        if hasattr(super(), "post_process"):
-            yield from super().post_process(paths, dry_run, **options)
-
         if dry_run:
             return
 
+        files = OrderedDict()
```

Comment:
I'm not clear why this file has to be changed, based on the pull request's description.

Reply:
look at super() class. with --dry-run no files are generated